Nasal/electrical.nas

```diff
@@ -500,7 +500,6 @@ var avionics_bus_1 = func() {
 var flapsDown = controls.flapsDown;
 controls.flapsDown = func(v) {
     flapsDown(v);
```

Comment:
It seems you can completely remove the overriding of `controls.flapsDown`. Without the `click` function it is pointless to override it.

Reply:
done